heron/api/src/scala/com/twitter/heron/streamlet/scala/converter/ScalaToJavaConverter.scala

```diff
@@ -32,9 +31,20 @@ object ScalaToJavaConverter {
       override def apply(r: R): T = f(r)
     }
 
+  def toSerializablePredicate[R](f: R => Boolean) =
+    new com.twitter.heron.streamlet.SerializablePredicate[R] {
```

Comment:
? why not import them? SerializablePredicate, SerializableConsumer, Sink, Context?

Reply:
@nwangtw Firstly, thanks for the review.

Updated by using multi import. Just an exceptional point here: Scala API supports own `Sink Trait` via `com.twitter.heron.streamlet.scala` package so `toJavaSink` function needs to have Java Sink package definition as follows due to using for same named Interface & Trait in different packages:
```
//ScalaToJavaConverter.scala

import com.twitter.heron.streamlet.scala.Sink

def toJavaSink[T](sink: Sink[T]): com.twitter.heron.streamlet.Sink[T]
```